src/AltinnCore/Common/Services/Implementation/ExecutionSIContainer.cs

```diff
@@ -211,9 +211,19 @@ public byte[] GetServiceResource(string org, string service, string resource)
         {
             byte[] fileContent = null;
 
-            if (File.Exists(_settings.BaseResourceFolderContainer + _settings.GetResourceFolder() + resource))
+            if (resource == "RuleHandler.js")
```

Comment:
"RuleHandler.js" is now hardcoded in many places, it would be better to use a constant like RuleHandlerFileName from ServiceRepositorySettings

Reply:
this to avoid stupid typo mistakes

Reply:
fixed 🍰